src/app/services/views/ServiceListView.vue

```diff
@@ -1,176 +1,146 @@
+              :headers="[
+                { label: 'Name', key: 'name' },
+                { label: 'Type', key: 'serviceType' },
+                { label: 'Address', key: 'addressPort' },
+                { label: 'Status', key: 'status' },
+                { label: 'DP proxies (online / total)', key: 'online' },
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ]"
+              :page-number="props.page"
+              :page-size="props.size"
+              :total="data?.total"
+              :items="data?.items"
+              :error="error"
+              @change="route.update"
+            >
+              <template #name="{ row: item }">
+                <RouterLink
+                  :to="{
+                    name: 'service-detail-view',
+                    params: {
+                      service: item.name,
+                    },
+                  }"
+                >
+                  {{ item.name }}
+                </RouterLink>
+              </template>
+              <template #status="{ row: item }">
+                <StatusBadge
+                  v-if="item.status"
+                  :status="item.status"
+                />
+
+                <template v-else>
+                  —
+                </template>
+              </template>
+
+              <template #online="{ row: item }">
+                <template
+                  v-if="item.dataplanes"
+                >
+                  {{ item.dataplanes.online || 0 }} / {{ item.dataplanes.total || 0 }}
+                </template>
+                <template
+                  v-else
+                >
+                  —
+                </template>
+              </template>
+              <template #actions="{ row: item }">
+                <KDropdownMenu
+                  class="actions-dropdown"
+                  :kpop-attributes="{ placement: 'bottomEnd', popoverClasses: 'mt-5 more-actions-popover' }"
+                  width="150"
+                >
+                  <template #default>
+                    <KButton
+                      class="non-visual-button"
+                      appearance="secondary"
+                      size="small"
+                    >
+                      <template #icon>
+                        <KIcon
+                          color="var(--black-400)"
+                          icon="more"
+                          size="16"
+                        />
+                      </template>
+                    </KButton>
+                  </template>
+                  <template #items>
+                    <KDropdownItem
+                      :item="{
+                        to: {
+                          name: 'service-detail-view',
+                          params: {
+                            service: item.name,
+                          },
+                        },
+                        label: 'View',
```

Comment:
**Suggestion**: Please move this into the message file. Also, it should be "View details" to be consistent with other views.

Reply:
Moved